src/feedspora/generic_client.py

```diff
@@ -199,12 +199,14 @@ def set_common_opts(self, account):
         '''
 
         # Tags
+        self._tags = []
```

Comment:
If `tags in account` is `True`, `self._tags` will be initialized by the list comprehension. If `tags in account` is `False`, then `self._tags` should be initialized with something compatible with its subsequent use in the code. The cleaner option would be not to change its default value (`None`) and handle it downstream in the code. However, an empty list is fine too. Its initialization would be clearer in an `else` statement, though.

Reply:
OK, but we need to work out the test functionality question below first, I think.

Reply:
Indeed. I saw your comment after I posted mine.